src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -552,6 +584,85 @@ def btrfs?
         def root?
           filesystem.root?
         end
+
+        # This implements the code that used to live in
+        # Yast::Filesystems::SuggestMPoints (whatever the rationle behind those
+        # mount paths was back then) with the only exception mentioned in
+        # {#booting_paths}
+        #
+        # @return [Array<String>]
+        def all_mount_paths
+          @all_mount_paths ||=
+            if Yast::Stage.initial
```

Comment:
Question: `#initial` means during installation?

Reply:
Yes. First stage (so installation or first phase of the autoinstallation)